packages/angular/src/generators/stories/stories.ts

```diff
@@ -35,11 +35,22 @@ export function angularStoriesGenerator(
     );
   }
 
+  const pathsToIgnore = options.ignorePaths?.split(',');
+  const { root } = readProjectConfiguration(tree, options.name);
+
   componentsInfo.forEach((info) => {
     if (info === undefined) {
       return;
     }
 
+    let skipPath = false;
+
+    pathsToIgnore?.forEach((ignorePath) => {
+      if (info.path.startsWith(`${root}/${ignorePath}`)) skipPath = true;
+    });
```

Comment:
The idea is that the user submits an array of paths, but these paths can be directory paths, right? So, they could say "Ignore all components under `src/lib/katerina`". I should check every visited path if it starts with `src/lib/katerina` right?

Reply:
Maybe have a look how the `inputs` glob is implemented in Nx core. I think it would be nice to have a consistent setup between the two and in general wherever we use globs for including/excluding things.

`inputs` also supports things like `{projectRoot}`.
 
Basically, the idea could be to have something like

```
"ignore": ["{projectRoot}/someFolder/**/*.component.ts"]
```

(we might want to think about a better name s.t. people immediately understand what it does)